distmv/statdist.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/gonum/stat"
 )
 
+type DistNormaler interface {
```

Comment:
We don't use them anywhere and they are very small. Remember that we culled a large number of interfaces from mat64 for these reasons.